### PR TITLE
Bug fix's!

### LINKED ISSUE #5245: You cannot eject the contents of an all-in-one grinder if it is not powered
Same issue as here: https://github.com/Aurorastation/Aurora.3/issues/4847
Except now the grinder. You should be able to take stuff out of the bowl/beaker

### LINKED ISSUE #5310: Feeding a slime water doesnt hurt them
If a slime is fed water with a breaker, they dont take damage.
If a slime is spashed, then they do.

### LINKED ISSUE #5322: Hand teles have no price assigned to them
Currently a Hand teleporter is worth only 5 credits, something more like 1000 would probably be more appropriate.

### LINKED ISSUE #5410: Advanced batteries don't fit into laptops
image
nuff said

### LINKED ISSUE #5412: NTOS copyright has expired
In modular computers, 
image

### LINKED ISSUE #5416: The Hydrogen loading KA loads with uranium.
The hydrogen loading KA loads in uranium, and I have a feeling this is a mistake. 
That or it's meant to mislead, in which case nice. Either way, it should be fixed to load mhydrogen or be renamed to uranium loading.

### LINKED ISSUE #5426: Briefcases don't show a sprite when facing the direction opposite of your hand.
Ideally, the briefcase should be partially visible despite being in the hand opposite of the 'camera' view.

Not visible while in left hand, facing right.
https://puu.sh/BL8oz/3a6e6d8209.png

Visible while in left hand, facing left.
https://puu.sh/BL8oY/85e2833b06.png

Not visible while in right hand, facing left
https://puu.sh/BL8pg/341fc44259.png

It just disappears.

### PR DESCRIPTION
**Pull request by dragoslava**
- rscadd: "Fixed Hand Teleporter Price being 5 credits, it is now 1800."
  - rscdel: "Made the grinder be able to eject its contents and beaker without power."
  - rscdel: "Slimes that drink water now get major toxin damage, far more then a shower."
Fixes #5426
Fixes #5322 
Fixes #5245 
Fixes #5310 
Fixes #5416 
Fixes #5412 
Fixes #5410

### CODE REVIEW

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Core.dm [hunk 1]
```diff
@@ -238,6 +238,18 @@
 		if(dose == removed)
 			S.visible_message("<span class='warning'>[S]'s flesh sizzles where the water touches it!</span>", "<span class='danger'>Your flesh burns in the water!</span>")
 
+
+/datum/reagent/water/affect_blood(var/mob/living/carbon/M, var/alien, var/removed)
+	if(istype(M, /mob/living/carbon/slime))
+		var/mob/living/carbon/slime/S = M
+		S.adjustToxLoss(20 * removed) // A slime having water forced down its throat would cause much more damage then being splashed on it
```
Comment (Arrow768) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Core.dm]: 20 damage per unit of water is excessive.

**Comment by Kaedwuff** [comment]
Wouldn't it be better to just prevent them from being forcefed water? They don't have mouths, it's a silly interaction.

File: code/modules/reagents/Chemistry-Machinery.dm [hunk 2]
```diff
@@ -809,3 +809,17 @@
...
 			spawn(10)
 			user.visible_message("<span class='warning'>[user] stops the [src] and leaves [target] resting as they are.</span>", "<span class='warning'>You turn the [src] off and let go of [target].</span>")
+
+/obj/machinery/reagentgrinder/verb/Eject()
+	set src in oview(1)
+	set category = "Object"
+	set name = "Eject contents"
+
+	usr.visible_message(
+	"<span class='notice'>[usr] open's [src] and has removed [english_list(contents)].</span>"
```
Comment (Alberyk) [inline code/modules/reagents/Chemistry-Machinery.dm]: I do not think that a visible message is needed here.

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Core.dm [hunk 3]
```diff
@@ -238,6 +238,18 @@
...
+		if(!S.client)
+			if(S.Target) // Like cats
+				S.Target = null
+				++S.Discipline
+		if(dose == removed)
+			S.visible_message("<span class='warning'>[S]'s screams in pain as you force water down its throat!</span>", "<span class='danger'>Your insides burn!</span>")
```
Comment (Alberyk) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Core.dm]: A visible message is also not needed here, I don't think that slimes even feel people or scream when in contact with water, at most, they just fizzle when it touches them.

File: code/modules/reagents/Chemistry-Machinery.dm (hunk 2 above)
Comment (skull132) [reply code/modules/reagents/Chemistry-Machinery.dm]: Not really the best or the worst, but:
* Fix the grammar a little -> `[usr] opens [src] and has removed [english_list(holdingitems)]`
* Do this text _after_ the `do_after` check.

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Core.dm (hunk 3 above)
Comment (skull132) [reply code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Core.dm]: This message cannot be used anyways. Since we cannot ensure that the slime is being forcefed from this proc. So, no visible message here.

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Core.dm [hunk 4]
```diff
@@ -238,6 +238,18 @@
 		if(dose == removed)
 			S.visible_message("<span class='warning'>[S]'s flesh sizzles where the water touches it!</span>", "<span class='danger'>Your flesh burns in the water!</span>")
 
+
+/datum/reagent/water/affect_blood(var/mob/living/carbon/M, var/alien, var/removed)
+	if(istype(M, /mob/living/carbon/slime))
+		var/mob/living/carbon/slime/S = M
+		S.adjustToxLoss(12 * removed) // A slime having water forced down its throat would cause much more damage then being splashed on it
+		if(!S.client)
```
Comment (skull132) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Core.dm]: this can be merged into `if (!S.client && S.Target)`

File: code/modules/reagents/Chemistry-Machinery.dm [hunk 5]
```diff
@@ -809,3 +809,17 @@
 			target.say("*scream")
 			spawn(10)
 			user.visible_message("<span class='warning'>[user] stops the [src] and leaves [target] resting as they are.</span>", "<span class='warning'>You turn the [src] off and let go of [target].</span>")
+
+/obj/machinery/reagentgrinder/verb/Eject()
```
Comment (Lohikar) [inline code/modules/reagents/Chemistry-Machinery.dm]: Missing sanity checks. Mice will be able to use this verb.

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Core.dm (hunk 3 above)
Comment (Lohikar) [reply code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Core.dm]: Grammar is invalid too.

File: code/modules/reagents/Chemistry-Machinery.dm (hunk 5 above)
Comment (Lohikar) [reply code/modules/reagents/Chemistry-Machinery.dm]: And ghosts, probably.

File: code/modules/reagents/Chemistry-Machinery.dm [hunk 6]
```diff
@@ -809,3 +809,20 @@
...
+	set src in oview(1)
+	set category = "Object"
+	set name = "Eject contents"
+
+	if (!ishuman(usr))
+		return
+
+	if (!do_after(usr, 1 SECONDS, act_target = src))
+		return
+		usr.visible_message(
```
Comment (skull132) [inline code/modules/reagents/Chemistry-Machinery.dm]: This and the following lines need to be tabbed back one.

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Core.dm [hunk 7]
```diff
@@ -238,6 +238,17 @@
 		if(dose == removed)
 			S.visible_message("<span class='warning'>[S]'s flesh sizzles where the water touches it!</span>", "<span class='danger'>Your flesh burns in the water!</span>")
 
+
+/datum/reagent/water/affect_blood(var/mob/living/carbon/M, var/alien, var/removed)
+	if(istype(M, /mob/living/carbon/slime))
+		var/mob/living/carbon/slime/S = M
+		S.adjustToxLoss(12 * removed) // A slime having water forced down its throat would cause much more damage then being splashed on it
+		if (!S.client && S.Target)
+			if(S.Target) // Like cats
```
Comment (skull132) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Core.dm]: Since you're already doing `&& S.Target` in the above if, you can remove this and tab the lines below it back one.

File: code/modules/modular_computers/hardware/battery_module.dm [hunk 8]
```diff
@@ -12,11 +12,10 @@
 
 /obj/item/weapon/computer_hardware/battery_module/advanced
 	name = "advanced battery"
-	desc = "An advanced power cell, often used in most laptops. It is too large to be fitted into smaller devices. It's rating is 1100."
```
Comment (Arrow768) [inline code/modules/modular_computers/hardware/battery_module.dm]: Why did you increase the battery rating ?

File: code/modules/custom_ka/cells.dm [hunk 9]
```diff
@@ -256,5 +256,5 @@
 
 	origin_tech = list(TECH_MATERIAL = 4,TECH_ENGINEERING = 5,TECH_MAGNET = 4,TECH_POWER = 4)
 
-	type_to_take = "uranium"
-	charge_per_sheet = 50
\ No newline at end of file
+	type_to_take = "mhydrogen"
+	charge_per_sheet = 90
```
Comment (BurgerLUA) [inline code/modules/custom_ka/cells.dm]: Why was charge per sheet changed?

File: code/modules/clothing/under/syndicate.dm [hunk 10]
```diff
@@ -18,6 +18,7 @@
 	item_state = "bl_suit"
 	worn_state = "tactifool"
 	siemens_coefficient = 1
+	has_sensor = 1
```
Comment (BurgerLUA) [inline code/modules/clothing/under/syndicate.dm]: The tacticool turtleneck isn't supposed to have sensors, this has been an intentional feature for a while due to the fact that the tacticool turtleneck gives 10 armor protection.

Comment (Lohikar) [reply code/modules/clothing/under/syndicate.dm]: This variant does not have armor.

Comment (dragoslava) [reply code/modules/clothing/under/syndicate.dm]: Essential what lok said. This is the civilian loadout version

**Comment by Lohikar** [comment]
You seem to have erased the majority of the changes in this PR.

**Comment by dragoslava** [comment]
> You seem to have erased the majority of the changes in this PR.

They are still there. I updated the branch so it caused it to be fucky @Sindorman confifmed it

**Comment by Lohikar** [comment]
No, they're not. I'm looking at the diff on here and you have only one change: a single line deleted on the map.

**Comment by Sindorman** [comment]
@Lohikar I have reverted things, everything is as it should be now. Let me know if it is off.

File: code/modules/reagents/Chemistry-Machinery.dm [hunk 11]
```diff
@@ -809,3 +809,17 @@
...
 			user.visible_message("<span class='warning'>[user] stops the [src] and leaves [target] resting as they are.</span>", "<span class='warning'>You turn the [src] off and let go of [target].</span>")
+
+/obj/machinery/reagentgrinder/verb/Eject()
+	set src in oview(1)
+	set category = "Object"
+	set name = "Eject contents"
+
+	if (!ishuman(usr))
+		return
+		usr.visible_message(
```
Comment (Lohikar) [inline code/modules/reagents/Chemistry-Machinery.dm]: This line is unreachable. Looks like skull's comment was marked resolved but the issue was not actually resolved.

File: code/modules/reagents/Chemistry-Machinery.dm [hunk 12]
```diff
@@ -809,3 +809,17 @@
 			target.say("*scream")
 			spawn(10)
 			user.visible_message("<span class='warning'>[user] stops the [src] and leaves [target] resting as they are.</span>", "<span class='warning'>You turn the [src] off and let go of [target].</span>")
+
+/obj/machinery/reagentgrinder/verb/Eject()
+	set src in oview(1)
+	set category = "Object"
+	set name = "Eject contents"
+
+	if (!ishuman(usr))
```
Comment (Lohikar) [inline code/modules/reagents/Chemistry-Machinery.dm]: This is insufficient sanity. Use `use_check()` instead of trying to roll your own checks.

File: nano/templates/ntnet_downloader.tmpl [hunk 13]
```diff
@@ -129,4 +129,4 @@
 {{/if}}
 <br><br>
 
-<div style="position: fixed; bottom: 5px;"><hr><i>NTOS v2.0.6 Copyright NanoTrasen 2450 - 2458</i></div>
\ No newline at end of file
+<div style="position: fixed; bottom: 5px;"><hr><i>NTOS v2.0.6 Copyright NanoTrasen 2450 - 2568</i></div>
```
Comment (Lohikar) [inline nano/templates/ntnet_downloader.tmpl]: I don't believe the current year is 2568.

File: code/modules/modular_computers/hardware/battery_module.dm [hunk 14]
```diff
@@ -15,7 +15,6 @@
 	desc = "An advanced power cell, often used in most laptops. It is too large to be fitted into smaller devices. It's rating is 1100."
 	icon_state = "battery_advanced"
 	origin_tech = list(TECH_POWER = 2, TECH_ENGINEERING = 2)
-	hardware_size = 3
```
Comment (Lohikar) [inline code/modules/modular_computers/hardware/battery_module.dm]: Removing this is going to let you put these into tablets, no?

File: nano/templates/ntnet_downloader.tmpl (hunk 13 above)
Comment (dragoslava) [reply nano/templates/ntnet_downloader.tmpl]: Thats when the copyright runs out

Comment (Sindorman) [reply nano/templates/ntnet_downloader.tmpl]: @Lohikar I believe that this needs to be greater then current year at least by 5-10 years.

Comment (Lohikar) [reply nano/templates/ntnet_downloader.tmpl]: The current year is 2460, btw. This is 108 years in the future, ICly.

Comment (dragoslava) [reply nano/templates/ntnet_downloader.tmpl]: Ok and by extending the copyright law out, we never have to worry about this issue again.

Comment (Lohikar) [reply nano/templates/ntnet_downloader.tmpl]: And it's invalid to future-date copyrights. IRL the copyright only can be updated when the relevant material is updated as well.

File: code/modules/modular_computers/hardware/battery_module.dm (hunk 14 above)
Comment (dragoslava) [reply code/modules/modular_computers/hardware/battery_module.dm]: fixed

File: nano/templates/ntnet_downloader.tmpl [hunk 15]
```diff
@@ -129,4 +129,4 @@
 {{/if}}
 <br><br>
 
-<div style="position: fixed; bottom: 5px;"><hr><i>NTOS v2.0.6 Copyright NanoTrasen 2450 - 2458</i></div>
\ No newline at end of file
+<div style="position: fixed; bottom: 5px;"><hr><i>NTOS v2.0.7 Copyright NanoTrasen 2460 - 2498</i></div>
```
Comment (Lohikar) [inline nano/templates/ntnet_downloader.tmpl]: This is still in the future. Do:
```suggestion
<div style="position: fixed; bottom: 5px;"><hr><i>NTOS v2.0.7 Copyright NanoTrasen 2450 - 2460</i></div>
```

Comment (Lohikar) [inline nano/templates/ntnet_downloader.tmpl]: Not that it matters, but why did you change this ver num?

File: code/modules/reagents/Chemistry-Machinery.dm [hunk 16]
```diff
@@ -809,3 +809,17 @@
...
+
+/obj/machinery/reagentgrinder/verb/Eject()
+	set src in oview(1)
+	set category = "Object"
+	set name = "Eject contents"
+
+	if(use_check(usr))
+		return
+	usr.visible_message(
+	"<span class='notice'>[usr] opens [src] and has removed [english_list(contents)].</span>"
```
Comment (skull132) [inline code/modules/reagents/Chemistry-Machinery.dm]:
```suggestion
	"<span class='notice'>[usr] opens [src] and has removed [english_list(holdingitems)].</span>"
```

Serializing contents is bad, mkay.
